CHANGES.rst

```diff
@@ -1,6 +1,27 @@
 Release Notes
 =============
 
+1.4.9
+-----
+
+This is a hotfix release for 1.4.8 that fixes a regression in interpreter setup that could lead to
+resolved distributions failing to build or install.
+
+* Support environment markers during pex activation. (#582)
```

Comment:
Hrm... wasn't thinking. This commit should not be in the release.
Thinking now how I want to handle.

Reply:
Alright - I guess the only real option is to just do 1.5.0. I'll bump the version and change the descriptions.

Reply:
Alright - fixed up. Sorry about this folks!